### PR TITLE
Update openmaptile

### PR DESCRIPTION
**Pull request by tbicr**
- update to work with latest openmaptile
- move openmaptile patch to separate repo for more easyest update
- update style: remove water moves and add housenubers
image
image

### CODE REVIEW

**Comment by abitrolly** [comment]
* explain what the `openmapfile` is

File: .gitmodules [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[submodule "openmaptiles"]
+	path = openmaptiles
+	url = https://github.com/tbicr/openmaptiles.git
```
Comment (Komzpa) [inline .gitmodules:3]: can you move this one to osm-by org?

Comment (tbicr) [reply .gitmodules:3]: Sound good for me, but I don't have permissions to do that

**Comment by tbicr** [comment]
>     * explain what the `openmapfile` is

You perhaps mean https://github.com/openmaptiles/openmaptiles - there are ecosystem to generate tiles: styles, data format, software to generate tiles

File: .gitmodules (hunk 1 above)
Comment (Komzpa) [reply .gitmodules:3]: check now?

Comment (tbicr) [reply .gitmodules]: moved

**Pull request merged**